test/extended/openstack/loadbalancers.go

```diff
@@ -165,7 +165,7 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb] The Openstack
 			o.Expect(err).NotTo(o.HaveOccurred())
 			maxSharedLb, err := strconv.Atoi(setting)
 			o.Expect(err).NotTo(o.HaveOccurred())
-			if 2 < maxSharedLb {
+			if 2 > maxSharedLb {
```

Comment:
Nit, `if maxSharedLb < 2` is easier to read and better matches that the error on the next line reports.